lib/widgets/action_sheet.dart

```diff
@@ -435,3 +616,30 @@ class ShareButton extends MessageActionSheetMenuItemButton {
     }
   }
 }
+
+class MessageActionSheetCancelButton extends StatelessWidget {
+  const MessageActionSheetCancelButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final designVariables = DesignVariables.of(context);
+    return TextButton(
+      style: TextButton.styleFrom(
+        padding: const EdgeInsets.all(10),
+        tapTargetSize: MaterialTapTargetSize.shrinkWrap,
+        minimumSize: Size.zero,
+        foregroundColor: designVariables.contextMenuCancelText,
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(7)),
+        splashFactory: NoSplash.splashFactory,
+      ).copyWith(backgroundColor: WidgetStateColor.resolveWith((states) =>
+          designVariables.contextMenuCancelBg.withOpacity(
+            states.contains(WidgetState.pressed) ? 0.20 : 0.15))),
```

Comment:
It's actually taken from Figma 🙂: https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=421-14113&t=gfnSxRpfQnio7zUP-0

Reply:
I see, thanks for pointing me to that! I was on a different page ("Main menu" instead of "Context menu"), but that page is also marked as ready for dev, and I agree it's a close enough match that it makes sense to use it here.

Reply:
Yeah, that (cancel) button is actually the "main component" of the button used in the "Context menu".
<img width="461" alt="temp figma" src="https://github.com/user-attachments/assets/75dcdce9-fabd-436d-8f1b-7493042b0277">

Reply:
💯